### PR TITLE
Changes done

### PR DESCRIPTION
**Pull request by alijawosti**
Made the addition of my greetings!

### CODE REVIEW

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -26,18 +26,22 @@ public String generateOutput() {
...
 
+  private String alijaSaysHello() {
+    return "Alija says 'Hey!'\n";
+  }
+
   private String ethanSaysHello() {
     return "Ethan says 'Yo!'\n";
   }
-    
+
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java:36]: Thanks for adding in these blank lines!

**Review by NicMcPhee — APPROVED** [review]
I assume you gave up on your other pull request and started over? That works, but only because the changes here were quite minor. If the changes had been more substantial, you would have wanted to have tried to make that other PR work.

**Review by LinneaMay — APPROVED** [review]
changes look good

File: src/main/java/hellos/Hellos.java (hunk 1 above)
Comment (alijawosti) [reply src/main/java/hellos/Hellos.java:36]: They were unintentional, sorry!

**Pull request merged**